Memory/OffsetInfo.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Sync.Tools;
+
+namespace OsuRTDataProvider.Memory
+{
+    public class OffsetInfo
+    {
+        //Version,BeatmapAddressOffset,BeatmapFolderAddressOffset,BeatmapSetAddressOffset,BeatmapFileNameAddressOffset,VersionCompareOperator
+        private const string VersionOffsetString = 
```

Comment:
please define a struct/class like : 
```C#
public struct VersionOffset
{
  public Version Version{get; set;}
  public int BeatmapAddressOffset{get; set;}
  public int BeatmapFolderAddressOffset{get; set;}
  public int BeatmapSetAddressOffset{get; set;}
  public int BeatmapFileNameAddressOffset{get; set;}
  public enum CompareCondition
{
  OlderOrEquals,
  Older,
  Equals,
  NewerOrEquals,
 Newer
}

  public CompareCondition VersionCompareCondition {get;set;}

public override string ToString() => $"{VersionCompareCondition} {Version} BeatmapFolder:{BeatmapAddressOffset} ....";
}
```

magic string is hard to read and debug for others.